### PR TITLE
Added greetings

### PR DESCRIPTION
**Pull request by singhsegv**

### CODE REVIEW

File: FUCKOFF_BY_SINGHSEGV.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Overview
+
+The motive of this file here is to gently say Fuck Off to the repo owner.
+
+## Author
+
+[Rajdeep Singh](https://github.com/singhsegv)
```
Comment (junaid1460) [inline FUCKOFF_BY_SINGHSEGV.md:7]: Guess you were hallucinating?

File: FUCKOFF_BY_SINGHSEGV.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Overview
+
+The motive of this file here is to gently say Fuck Off to the repo owner.
```
Comment (junaid1460) [inline FUCKOFF_BY_SINGHSEGV.md:3]: Yup, got that

File: FUCKOFF_BY_SINGHSEGV.md (hunk 1 above)
Comment (singhsegv) [reply FUCKOFF_BY_SINGHSEGV.md:7]: After party kudiye?

Comment (junaid1460) [reply FUCKOFF_BY_SINGHSEGV.md:7]: lawde ke baal

**Pull request closed without merging**